.github/workflows/ubuntu_x86_64.yml

```diff
@@ -34,7 +34,10 @@ jobs:
         run: cd crates/compiler/builtins/bitcode && ./run-wasm-tests.sh
 
       - name: regular rust tests
-        run: cargo test --locked --release --features with_sound serde --workspace && sccache --show-stats
+        run: cargo test --locked --release --features with_sound serde && sccache --show-stats
+
+      - name: check that the platform`s produced dylib is loadable
+        run: cd examples/platform-switching/rust-platform && LD_LIBRARY_PATH=. cargo test --release --locked
```

Comment:
@Anton-4  this still needs the `roc gen-stub-lib` invocation right?

Reply:
I may be misremembering but I believe you mentioned this was unnecessary if the example had run before.
However in this case, the tests did not run in the step before, I think it's because of the features flag. I'm going to try removing it.